attack_utilities/dp_svc_example.py

```diff
@@ -0,0 +1,113 @@
+'''
```

Comment:
Not sure we need this code? And if we do, it would find a more natural home in the examples folder?